crates/language_server/src/analysis/tokens.rs

```diff
@@ -771,7 +771,7 @@ impl IterTokens for Loc<Pattern<'_>> {
             Pattern::Identifier { .. } => onetoken(Token::Variable, region, arena),
             Pattern::Tag(_) => onetoken(Token::Tag, region, arena),
             Pattern::OpaqueRef(_) => onetoken(Token::Type, region, arena),
-            Pattern::Apply(p1, p2) => (p1.iter_tokens(arena).into_iter())
+            Pattern::Apply(p1, p2, _) => (p1.iter_tokens(arena).into_iter())
```

Comment:
Not 100% sure, but this may need to do something with the parens and commas. I'd check if we output tokens for parens and commas elsewhere.